color_width: 640
color_height: 480
color_fps: 30
color_qos: "SYSTEM_DEFAULT"

depth_width: 640
depth_height: 480
depth_fps: 30
depth_qos: "SENSOR_DATA"
depth_info_qos: "SYSTEM_DEFAULT"
